pom.xml

```diff
@@ -195,8 +195,8 @@
     </pluginManagement>
   </build>
   <modules>
-    <module>test/vanilla</module>

```

Comment:
This is a change from autorest.java that you probably want but it just got trampled in the replay.